### PR TITLE
Add vista3d bundle

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes # .

### Description
Add Vista3d Bundle and update tests

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/vista3d/msd_task09_spleen_folds.json
Comment (KumoLiu) [inline models/vista3d/msd_task09_spleen_folds.json:1]: should we put this json under config folder or on the google drive?

Comment (yiheng-wang-nv) [reply models/vista3d/msd_task09_spleen_folds.json:1]: thanks, moved to `configs`

File: models/vista3d/docs/data_license.txt
Comment (KumoLiu) [inline models/vista3d/docs/data_license.txt:1]: Given that we will have multiple license files, should we also unify the format of this license for data?
cc @Nic-Ma @yiheng-wang-nv

File: models/vista3d/configs/data.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# this file shows specific settings for evaluation on each dataset
+validate#postprocessing#transforms#0#_disabled_: false
+validate#handlers#2#_disabled_: true
+validate#evaluator#key_val_metric: null
+eval_folder: "/eval/"
+ckpt_path: "$@bundle_root + '/models/model.pt'"
+label_set: "$list(set([i+1 for i in range(117)]) - set([22, 23, 15, 25, 19, 2, 26, 27, 28, 29, 117]))"
+val_label_set: "$list(range(118))"
+label_mappings: "${}"
+# label_mappings:
```
Comment (KumoLiu) [inline models/vista3d/configs/data.yaml:10]: May I ask why we comment the default mapping here?

Comment (yiheng-wang-nv) [reply models/vista3d/configs/data.yaml:10]: It comes from the author side, I did not change here.

File: ci/verify_bundle.py [hunk 2]
```diff
@@ -65,52 +58,32 @@ def _get_net_id(bundle: str):
...
-def _check_main_section_optional_key(
-    arg_name: str, necessary_key: str, config: dict, main_section: str = "train", sub_section: str = "trainer"
-):
-    # if `arg_name` is in `sub_section`, its value must be `necessary_key`
-    if arg_name in config[main_section][sub_section]:
-        if necessary_key not in config[main_section]:
-            actual_key = str(config[main_section][sub_section][arg_name]).split("#")[-1]
-            raise ValueError(f"'{main_section}' should have '{necessary_key}', got '{actual_key}'.")
 
+def _find_license_file(bundle_path: str):
```
Comment (KumoLiu) [inline ci/verify_bundle.py:75]: Do we still need this function here?

Comment (KumoLiu) [reply ci/verify_bundle.py:75]: discussed offline, we can leave it here.

Comment (yiheng-wang-nv) [reply ci/verify_bundle.py:75]: discussed offline

**Review by KumoLiu — APPROVED** [review]

**Pull request merged**